libexec/helpers/image-import.sh

```diff
@@ -54,7 +54,7 @@ fi
 case "$IMPORT_URI" in
     docker://*)
         CONTAINER_NAME=`echo "$IMPORT_URI" | sed -e 's@^docker://@@'`
-        SINGULARITY_IMPORT_GET="$SINGULARITY_libexecdir/singularity/python/cli.py --rootfs '$SINGULARITY_ROOTFS' --docker '$CONTAINER_NAME' --cmd"
+        SINGULARITY_IMPORT_GET="$SINGULARITY_libexecdir/singularity/python/cli.py --rootfs '$SINGULARITY_ROOTFS' --docker '$CONTAINER_NAME' ${SINGULARITY_DOCKER_REGISTRY:-} ${SINGULARITY_DOCKER_AUTH:-} --cmd"
     ;;
```

Comment:
I didn't look at this part when @gmkurtzer implemented - it looks like it doesn't include logic for including / not including the command as runscript. Right now, the default (without `--cmd`) is to use the entrypoint, which is the more proper of the two (CMD and ENTRYPOINT) as it is the container's executable. Adding the `--cmd` argument overrides this default, and changes the runscript to be CMD. For import, we would likely want the default to be the same (not having CMD) and akin to the docker credentials, have it be an envrionmental variable. Thoughts @gmkurtzer @flx42 ?